### DISCUSSION TITLE
Peas

### DISCUSSION THREAD

**Discussion by joe042** [community/community#35669] [Other Feature Feedback, Questions, & Ideas]
Hello guys please I want someone will explain this Repositor

**Comment by jostasik** [comment]
This community/community repository was chosen to be the home for discussions that are organization-wide in scope.  
It's being treated as the official platform for GitHub Q&A, show & tell, and feedback on GitHub. 
It's just a nice place for everyone to be able to send feedback to the staff members and get help from the broader community.

**Comment by jfullstackdev** [comment]
GitHub Discussion is an added feature, before there was the Issues tab that you can use to discuss things, now they released another great feature, the GitHub Discussion and since it's their product, they are using it now, before this, GitHub Community was using an external one for the forum

you might be confused because it's in a repo, and repos are originally intended to be for code but because of GitHub Discussion, you can now host even your own forum right here on GitHub, that's the power of GitHub Discussion and one ultimate example of that being used as a forum is this GitHub Community itself

**Reply by Kavex71** [reply]
no si sea aqui donde pueda  genarar un ssh , escribe el texto que me dijieron en la terminal pero no hace nada nomas me que  no es un comando, ahora tambien alguien me decir como pegar un texto en mi correo electronico
